docs/source/install-windows.rst

```diff
@@ -0,0 +1,44 @@
+.. _install-windows:
+
+######################
+Installation - Windows
+######################
+
+.. important:: THIS IS THE COZMO SDK BETA. The SDK is under development and is subject to change.
+
+This guide provides instructions on installing the Cozmo SDK for computers running with a Windows operating system.
+
+-------------------
+Python Installation
+-------------------
+
+Download the `Python 3.5.1 (or later) executable file from Python.org <https://www.python.org/downloads/>`_ and
+run it on your computer. We recommend that you tick the "Add Python 3.5 to PATH" checkbox on the Setup screen.
```

Comment:
The 'We recommend' sentence is very important. Could we bold it or make it red or something?